packages/rum-core/src/performance-monitoring/span-base.js

```diff
@@ -50,7 +50,7 @@ class SpanBase {
     this.traceId = options.traceId
     this.sampled = options.sampled
     this.timestamp = options.timestamp
-    this._start = now()
+    this._start = mayBeTime(options.startTime)
```

Comment:
Not sure if we need this function here. Why not just a simple falsy check on `options.startTime`.

Also, maybe just use `options.start` instead!

Reply:
have to account for 0 as well and was using it on lots of places so moved it to function.

Reply:
I think something like `isValidTime` is more clear about what is does and can be used elsewhere as well. `mayBeTime` is really ambiguous about what it actually does!

Also I'm questioning whether we should do any validation here. The reason is that if the value of `startTime` is not valid (e.g. negative value) currently we just use `now()` instead. This might prevent us from capturing bugs or could make debugging issues harder because we use `now()` when `startTime` is invalid.

Reply:
decided to rename the function to `getTime`